### DISCUSSION TITLE
Как получить id строк выбранных галочками (insert из linktotable выбранного)

### DISCUSSION THREAD

**Discussion by Alexmeb** [totumonline/totum-ru-issues-and-discussions#208] [Как это сделать?] [answered]
Не нашел, может плохо искал. А возможно ли делать код действия не только добавить и удалить, но и insert в другую таблицу полей из выбранной строки?
![Screenshot_36](https://user-images.githubusercontent.com/109726405/187278133-4e90e842-b188-4afd-9068-3b900e188729.png)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Есть переменная доступная только в кодах действия Кнопок.

+ `$#ids` — возвращает список id строк, выбранных галочками (доступна только в коде-действия Кнопок).

Логично использовать вместе с параметром `Снимать галочки после выполнения`.

**Reply by Alexmeb** [reply]
Да, переменную видел, не совсем понятно как её использовать в insertlist. У неё есть переменная куда добавятся id вставленных строк, но не выбранных галочкой...

**Reply by erpic-ru** [reply]
сначала `selectrowlist(table $#ntn; fields: "*ALL*" where: 'id' = $#ids)`, а потом уже insertlist...

**Reply by Alexmeb** [reply]
Ну да, делал же уже такое) Разобрался))
